packages/cozy-sharing/src/index.jsx

```diff
@@ -380,3 +380,16 @@ export const ShareModal = ({ document, ...rest }) => (
     }
   </SharingContext.Consumer>
 )
+/**
+ * Expose a refresh method
+ */
+export const RefreshableSharings = ({ children, ...rest }) => (
+  <SharingContext.Consumer>
+    {({ refresh }) =>
+      children({
+        rest,
```

Comment:
Did you mean `...rest`?
Also, I've never seen this, calling `children` as a function. Is there any doc about it that I missed? I only found [this](https://reactjs.org/docs/jsx-in-depth.html#functions-as-children) and there seems to be no guarantee that `children` is actually a function..?

Reply:
We already have `children()` in https://github.com/cozy/cozy-libs/blob/master/packages/cozy-sharing/src/components/SharedDocuments.jsx#L10 

It's just a `Function as Child Component` and it works well also with class component https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/drive/SharingsContainer.jsx#L171 

https://medium.com/merrickchristensen/function-as-child-components-5f3920a9ace9

In fact, I've no need of `rest`thanks :)